### PR TITLE
JBDS-3605 JBDS-3878 fix branding & versioning

### PR DESCRIPTION
**Pull request by nickboldt**
JBDS-3605 fix branding (no longer JBoss)
JBDS-3878 fix versioning s/9.1/10.0/g

### CODE REVIEW

File: browser/model/jbds.js [hunk 1]
```diff
@@ -18,8 +18,8 @@ import Util from './helpers/util';
 class JbdsInstall extends InstallableItem {
   constructor(installerDataSvc, downloadUrl, installFile, targetFolderName) {
     super('jbds',
-          'RED HAT JBOSS DEVELOPER STUDIO',
-          '9.1.0',
+          'RED HAT DEVELOPER STUDIO',
```
Comment (ppitonak) [inline browser/model/jbds.js]: This is the only component with name in upper case, is there any reason for that?

Comment (nickboldt) [inline browser/model/jbds.js]: It was like that when I found it, I swear. :D

Comment (dgolovin) [inline browser/model/jbds.js]: Should be "Red Hat Developer Studio" the same way as other components,

**Comment by nickboldt** [comment]
Replaced by https://github.com/redhat-developer-tooling/developer-platform-install/pull/189

**Pull request closed without merging**